### PR TITLE
Built in Pop-Up tower

### PR DESCRIPTION
**Pull request by JT122406**
Builds in the Pop-Up towers directly into Bedwars so Addon is not needed

### CODE REVIEW

File: versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java [hunk 1]
```diff
@@ -58,7 +58,6 @@ public VersionCommon(VersionSupport versionSupport) {
             }
 
             // 1.12 drop listeners
-            //if (versionSupport.getVersion() == 5){
```
Comment (andrei1058) [inline versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java]: why you doing that? that check is here for a reason

File: bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java [hunk 2]
```diff
@@ -770,7 +770,7 @@ public static Material getWool(@NotNull TeamColor teamColor) {
      *
      * @return 1.13+ material.
      */
-    public Material woolMaterial() {
+    public org.bukkit.Material woolMaterial() {
```
Comment (andrei1058) [inline bedwars-api/src/main/java/com/andrei1058/bedwars/api/arena/team/TeamColor.java]: just import it to keep it clean

**Review by andrei1058 — CHANGES_REQUESTED** [review]
Seems ok overall, but we need to find a cleaner way to store block positions.

**Comment by samupro-dev** [comment]
Add sounds can be configurable in `sounds.yml`

File: versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java (hunk 1 above)
Comment (JT122406) [reply versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java]: That was accident my bad

Comment (JT122406) [reply versionsupport_common/src/main/java/com/andrei1058/bedwars/support/version/common/VersionCommon.java]: I went looking for something in that class I think I went to leave a comment but then I accidently commented it out

**Comment by JT122406** [comment]
> Seems ok overall, but we need to find a cleaner way to store block positions.

Yeah I was doing what the addon did when I have time I will rewrite it to be more efficient and less weird with all those position

**Comment by Eradiator** [comment]
Nice... Now no need for a million addons lol

**Comment by andrei1058** [comment]
I'm looking forward for these changes

**Comment by JT122406** [comment]
Todo: Some 1.8.8 Issues have been reported will look into it, 
Still need All other Languages

**Comment by JT122406** [comment]
I fixed the issues with it

> Todo: Some 1.8.8 Issues have been reported will look into it, Still need All other Languages

**Comment by Eradiator** [comment]
@andrei1058  please update the plugin because its loosing popularity because of no updates

**Pull request merged**